### PR TITLE
UI Form: 31701, display error-message in form

### PR DESCRIPTION
**Pull request by nhaagen**
A user my add contraints on an entire form; however, there will not be any messages. 
See https://mantis.ilias.de/view.php?id=31701

This will display an (additional) error-message on forms, not only on inputs.

### CODE REVIEW

**Comment by Amstutz** [comment]
Hi @nhaagen

Your proposal looks GTM. Thx for adding test, to prevent this issue in future. Special thx for adding the role to the role alert to the output. Note that we have a change on the public interface, meaning, you or @rklees will have to quickly announce this to the next JF.

Thx a lot!
Timon

File: tests/UI/Component/Input/Container/Form/StandardFormTest.php [hunk 1]
```diff
@@ -136,4 +138,127 @@ public function test_render_no_url()
         ');
         $this->assertHTMLEquals($expected, $html);
     }
+
+
+    public function testRenderWithErrorOnFieled()
```
Comment (tfamula) [inline tests/UI/Component/Input/Container/Form/StandardFormTest.php]:
```suggestion
    public function testRenderWithErrorOnField()
```

Just a little typo

**Comment by matthiaskunkel** [comment]
Jour Fixe, 13 DEC 2021 : RIchard notified us about a change in a public UI interface. We highly appreciate this change for ILIAS 7 and trunk.

**Comment by Amstutz** [comment]
@nhaagen thx for contributing. LGTM. @tfamula thx for also looking into this!

@klees all lights green for a merge. feel free to do so or ping me if I should.

**Pull request merged**